salt/states/pip_state.py

```diff
@@ -440,9 +440,28 @@ def installed(
             - require:
               - pkg: python-pip
 
-    This will install the latest Django version greater than 1.6 but less
+    Installs the latest Django version greater than 1.6 but less
     than 1.7.
```

Comment:
Wouldn't you want to indent this whole section, starting with `Example` so that it falls under the `name` argument? It's just weird to see this indentation and then see the `pkgs` parameter right after.

Reply:
This is the current state of the documentation: https://docs.saltproject.io/en/latest/ref/states/all/salt.states.pip_state.html#salt.states.pip_state.installed